code/modules/mob/living/basic/pets/parrot/_parrot.dm

```diff
@@ -70,7 +70,7 @@ GLOBAL_LIST_INIT(strippable_parrot_items, create_strippable_list(list(
 		/obj/structure/frame/computer,
 	))
 	///items we wont pick up
-	var/static/list/ignore_items = typecacheof(list(/obj/item/radio))
+	var/static/list/ignore_items = typecacheof(list(/obj/item/radio, /obj/effect/name_tag)) //Monkestation addition: ignore their nametag
```

Comment:
tbh should just include all of `/obj/effect`

Reply:
Alright, done